main.cpp

```diff
@@ -203,13 +277,17 @@ void SubwooferFactory::purchasePlywood(int sheets, float thickness)
     {
         plywoodInStock += (sheets * 8.f);
     }
+    {
+        std::cout << "SubwooferFactory::purchasePlywood() 3/4\" plywood is needed! " << thickness << " is not a valid thickness." << std::endl;
+    }
```

Comment:
This expression does not need to be nested inside curly braces.
remove the curly braces and fix the indentation
- [ ] revise